sitewit/services.py

```diff
@@ -398,19 +419,23 @@ def create_partner(self, name, address, settings):
                 modelName=Address%20%28Create%29
             settings (dict): WL settings.
                 https://sandboxpapi.sitewit.com/Help/ResourceModel?modelName=
-                White%20Label%20Settings%20%28Create%29
+                White%20Label%20Settings%20%28Create%29,
+            remote_id (str): remote id of subpartner
 
         Returns:
             Please see response specification here:
             https://sandboxpapi.sitewit.com/Help/Api/GET-api-Partner
         """
-        data = {'name': name,
-                'address': address,
-                'whiteLabelSettings': settings}
+        data = {
+            'name': name,
+            'address': address,
+            'whiteLabelSettings': settings,
+            'remote_id': remote_id
```

Comment:
I didn't see this parameter mentioned in the ticket or on the browseable API docs. Did I miss something?